packages/trpc/server/routers/viewer/organizations/bulkDeleteUsers.handler.ts

```diff
@@ -57,13 +57,29 @@ export async function bulkDeleteUsersHandler({ ctx, input }: BulkDeleteUsersHand
     },
   });
 
+  const removeManagedEventTypes = prisma.eventType.deleteMany({
+    where: {
+      userId: {
+        in: input.userIds,
+      },
+      team: {
+        OR: [
+          {
+            parentId: currentUser.organizationId,
+          },
+          { id: currentUserOrgId },
+        ],
+      },
+    },
+  });
+
```

Comment:
In the issue it is mentioned that you need to only remove the assignment but here you are deleting all event types